transaction/std/src/transaction_builder.rs

```diff
@@ -67,6 +67,9 @@ pub struct TransactionViewOnlySigningData {
 
     /// rings
     pub signing_data: ViewOnlySigningData,
+
+    /// Output secrets
+    pub output_secrets: Vec<OutputSecret>,
```

Comment:
Do you think there's an issue including this here? It doesn't seem problematic to me since the blinding already ends up in `ViewOnlySigningData`, and supposedly the `Amount` is not secret anyway since it will need to be provided again when signing

Reply:
Why if you can just reconstruct it from the amounts and blindings?

Reply:
Just for convenience?

Reply:
I must've missed the amounts...
Can you please fix that? I am deep in 3 way merge cherrypicking hell

Reply:
Yeah for sure :)

Reply:
Oh wait I may be wrong about that

Reply:
Yeah okay, I guess OutputSecrets needs to be included if we want to preserve the output information for the user? Though it shouldn't matter from the perspective of signing and submitting the transaction, right?

Reply:
I dont think so, since at that point the information has been ingested into outputs living in TxPrefix